File: 2015/01/part1.py
import os
path = os.path.dirname(os.path.abspath(__file__))

with open(path+'/input.txt', 'r') as input:
	counter = 0
	text = input.readline()
	for char in text :
		if char == ')':
			counter -= 1
		elif char == '(':
			counter += 1
	print (counter)

File: 2015/05/part1.js
const fs = require('fs');
const input = fs.readFileSync('input.txt', 'utf-8').split('\n');

let counter = 0;
for (let str of input) {
	if ((str.match(/(a|e|i|o|u)/g) || []).length >= 3 &&
		str.match(/(\w)\1/g) &&
		!str.match(/(ab|cd|pq|xy)/))
		counter++;
}
console.log(counter)
